test/multi_dim_bc_test.jl

```diff
@@ -83,12 +88,15 @@ for N in 2:7
     sizes = rand(4:7, N)
     A = rand(sizes...)
 
-    Q1_N = Neumann0BC(Float64, Tuple(ones(N)), 3.0, size(A))
+    Q1_N = RobinBC(Tuple(rand(3)), Tuple(rand(3)), fill(0.1, N), 4.0, size(A))
```

Comment:
Is this syntax correct?  I don't think the size argument can go into the parameter

Reply:
this is the signature required for a multi dim bc, yeah